db/migrate/20230929163207_add_locked_to_user_verifications.rb

```diff
@@ -0,0 +1,5 @@
+class AddLockedToUserVerifications < ActiveRecord::Migration[6.1]
+  def change
+    add_column :user_verifications, :locked, :boolean, default: false
```

Comment:
Since we're able to add a default, we can also add a `null: false` index to this